CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 ## Unreleased
+* Added web3 providers supports such as http and ws protocols
+* Introduced ConfigurationError which handles all error that were made cause of falty configurations
```

Comment:
I believe that errors can't handle other errors (lexical mistake)

```suggestion
* Introduced ConfigurationError for validating method arguments
```
Still dunno about this exact wording, might sound a bit too abstract

Reply:
You need to describe why it is important to the user, like:

```suggestion
* Throw `ConfigurationError` instead of basic `Error` when Resolution library is configured incorrectly
```

In this way I know what I can do with it: I can target it specifically within `catch` block. That should be obvious, but JS developers are not that profound in error handling so we can even suggest that in a changelog.